packit_service/worker/reporting.py

```diff
@@ -120,15 +124,29 @@ def report(
         state: BaseCommitStatus,
         description: str,
         url: str = "",
+        links_to_external_services: Optional[Dict[str, str]] = None,
         check_names: Union[str, list, None] = None,
     ) -> None:
         """
-        set commit check status
+        Set commit check status.
+
+        Args:
+            state: State accepted by github.
+            description: The long text.
+            url: Url to point to (logs usually).
+
+                Defaults to empty string
+            links_to_external_services: Direct links to external services.
+                e.g. {"Testing Farm": "url-to-testing-farm"}
```

Comment:
```suggestion
                e.g. `{"Testing Farm": "url-to-testing-farm"}`
```

Google style docstrings support markdown.